torchdr/neighbor_embedding/sne.py

```diff
@@ -208,7 +208,7 @@ def __init__(
         perplexity=10,
         n_components=2,
         optimizer="Adam",
-        optimizer_kwargs=None,
+        optimizer_kwargs={},
```

Comment:
@huguesva  beware of having arguments that are mutable : 
![image](https://github.com/TorchDR/TorchDR/assets/8993218/b31dc1ca-3e7d-48eb-9b7e-dbeda5853fed)